src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -77,19 +82,24 @@ class PropertyAccessor implements PropertyAccessorInterface
      *                               or self::DISALLOW_MAGIC_METHODS for none
      * @param int $throw             A bitwise combination of the THROW_* constants
      *                               to specify when exceptions should be thrown
+     * @param int-mask-of<self::BYPASS_PROPERTY_HOOK_*> $bypassPropertyHooks A bitwise combination of the BYPASS_PROPERTY_HOOK_* constants
+     *                               to specify the hooks you want to bypass,
+     *                               or self::BYPASS_PROPERTY_HOOK_NONE for none
```

Comment:
looks like the previous arguments could also nicely benefit from similar int-mask-of 
let's align them also

the text looks redundant to me:
```suggestion
     * @param int-mask-of<self::BYPASS_PROPERTY_HOOK_*> $bypassPropertyHooks
```